estate/views/estate_menus.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem id="estate_property_menu_root" name="Estate Property">
+        <menuitem id="estate_property_first_level_menu" name="Advertisement">
+            <menuitem id="estate_property_model_menu_action" name="Properties" action="estate_property_model_action"/>
```

Comment:
```suggestion
            <menuitem id="action_estate_property_model_menu" name="Properties" action="action_estate_property_model"/>
```
Action names must start with `action_` followed by what it performs. Make sure to check all the implementations